### PR TITLE
Fixes to style guide, design (sumo.scss), README.md

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Fixes to style guide, nav and button design (sumo.scss), README.md

When I did the Docusaurus 3.4.0 update, it changed the font color on some top nav items. This PR turns those back to the correct colors.

Screenshot 2024-06-03 at 4 28 08 PM

I also changed the "start free trial" and "contact support" button design so that they pop more and look cleaner.

## Select the type of change
<!-- What types of changes does your code introduce? Select the checkbox after clicking "Create pull request" button. --

- [x] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Contributing
 
-Please see the [Contributor Guidelines](https://help.sumologic.com/docs/contributing) published on our docs site.
+Please see our [Contributor Guidelines](https://help.sumologic.com/docs/contributing).
```
Comment (kimsauce) [inline CONTRIBUTING.md:3]:
```suggestion
Please see the [Contributor Guidelines](https://help.sumologic.com/docs/contributing) published on our docs site.
```

File: README.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ Documentation staff members review issues and pull requests on a regular basis.
 
 ## Contributing Content
 
-For detailed guidelines, see [Contributor Guidelines](/docs/contributing).
+For detailed instructions, including our style guide, see [Contributor Guidelines](/docs/contributing).
```
Comment (kimsauce) [inline README.md:17]:
```suggestion
For detailed instructions, including our style guide, see [Contributor Guidelines](https://help.sumologic.com/docs/contributing).
```

File: README.md [hunk 3]
```diff
@@ -14,9 +14,9 @@ Documentation staff members review issues and pull requests on a regular basis.
 
 ## Contributing Content
 
-For detailed guidelines, see [Contributor Guidelines](/docs/contributing).
+For detailed instructions, including our style guide, see [Contributor Guidelines](https://help.sumologic.com/docs/contributing).
```
Comment (kimsauce) [inline README.md:17]: Needs to be the absolute link - this is the README

File: README.md [hunk 4]
```diff
@@ -14,9 +14,9 @@ Documentation staff members review issues and pull requests on a regular basis.
...
 
-We recommend forking the repo, creating all content changes in branches, and submitting pull requests. We will work with you to get this content reviewed, tested, and merged for publishing.
+We recommend forking our repo, applying your content changes in a new branch, and submitting a pull request. We will work with you to get this content reviewed, tested, and merged for publishing.
```
Comment (kimsauce) [inline README.md:19]:
```suggestion
We recommend you fork our repo, create a new branch, apply your content changes to that branch, and then submit a pull request. We will work with you to get this content reviewed, tested, and merged for publishing.
```

File: README.md [hunk 5]
```diff
@@ -14,9 +14,9 @@ Documentation staff members review issues and pull requests on a regular basis.
 
 ## Contributing Content
 
-For detailed guidelines, see [Contributor Guidelines](/docs/contributing).
+For detailed instructions, including our style guide, see [Contributor Guidelines](https://help.sumologic.com/docs/contributing).
 
-We recommend forking the repo, creating all content changes in branches, and submitting pull requests. We will work with you to get this content reviewed, tested, and merged for publishing.
+We recommend you fork our repo, create a new branch, apply your content changes to that branch, and then submit a pull request. We will work with you to get this content reviewed, tested, and merged for publishing.
```
Comment (kimsauce) [inline README.md:19]:
```suggestion
We recommend forking our repo, creating a new branch for your content changes, and submitting a pull request. We will help review, test, and merge the content for publishing.
```
```suggestion
We recommend you fork our repo, create a new branch, apply your content changes to that branch, and then submit a pull request. We will work with you to get this content reviewed, tested, and merged for publishing.
```

File: docs/contributing/style-guide.md [hunk 6]
```diff
@@ -1131,63 +1175,63 @@ We use the Oxford (serial) comma. For example, use "I had eggs, toast, and orang
...
 
 ### Text only
 
-To add release notes without images:
+To add a text-only release note:
 
-1. In the blog folder, create a new markdown file with the following name format: `YYYY-MM-DD-product.md`.
-1. Add the following frontmatter, swapping out these example values with your own.
+1. In the appropriate blog folder ([blog-collector](https://github.com/SumoLogic/sumologic-documentation/tree/main/blog-collector), [blog-cse](https://github.com/SumoLogic/sumologic-documentation/tree/main/blog-cse), [blog-csoar](/release-notes-csoar/), [blog-developer](https://github.com/SumoLogic/sumologic-documentation/tree/main/blog-developer), [blog-service](https://github.com/SumoLogic/sumologic-documentation/tree/main/blog-service)), add a new file that follows the format of the other posts in that folder. For example, for blog-service release notes, the format is `YYYY-MM-DD-<product/feature>`. For Cloud SIEM and SOAR, the format is `YYYY-MM-DD-application-update` or `YYYY-MM-DD-content-update`.<br/><img src={useBaseUrl('img/contributing/release-notes-dropdown-menu.png')} alt="Release notes menu" style={{border: '1px solid gray'}} width="200"/>
```
Comment (jpipkin1) [inline docs/contributing/style-guide.md:1184]:
```suggestion
1. In the appropriate blog folder ([blog-collector](https://github.com/SumoLogic/sumologic-documentation/tree/main/blog-collector), [blog-cse](https://github.com/SumoLogic/sumologic-documentation/tree/main/blog-cse), [blog-csoar](https://github.com/SumoLogic/sumologic-documentation/tree/main/blog-csoar), [blog-developer](https://github.com/SumoLogic/sumologic-documentation/tree/main/blog-developer), [blog-service](https://github.com/SumoLogic/sumologic-documentation/tree/main/blog-service)), add a new file that follows the format of the other posts in that folder. For example, for blog-service release notes, the format is `YYYY-MM-DD-<product/feature>`. For Cloud SIEM and SOAR, the format is `YYYY-MM-DD-application-update` or `YYYY-MM-DD-content-update`.<br/><img src={useBaseUrl('img/contributing/release-notes-dropdown-menu.png')} alt="Release notes menu" style={{border: '1px solid gray'}} width="200"/>
```

File: docs/contributing/style-guide.md [hunk 7]
```diff
@@ -1131,63 +1175,63 @@ We use the Oxford (serial) comma. For example, use "I had eggs, toast, and orang
...
     keywords:
       - alerts
+    image: https://help.sumologic.com/img/sumo-square.png
     authors:
       - url: https://help.sumologic.com/release-notes-service/rss.xml
         image_url: /img/release-notes/rss-orange.png
     ---
     ```
-    * `title`. Title for release note.
+    * `title`. Title for release note that includes Product or Feature name.
```
Comment (jpipkin1) [inline docs/contributing/style-guide.md:1198]:
```suggestion
    * `title`. Title for release note that includes product or feature name.
```

File: docs/contributing/style-guide.md [hunk 8]
```diff
@@ -1131,63 +1175,63 @@ We use the Oxford (serial) comma. For example, use "I had eggs, toast, and orang
...
        :::note
        For Service Release Notes only, append the title with the category name in parenthesis (example: `Automatic Log Level Detection (Search)`). See previous Service Release Notes for category names.
```
Comment (jpipkin1) [inline docs/contributing/style-guide.md:1200]:
```suggestion
       For service release notes only, append the title with the category name in parenthesis (example: `Automatic Log Level Detection (Search)`). See previous service release notes for category names.
```

**Review by jpipkin1 — DISMISSED** [review]
@kimsauce, I don't see the Google Translate widget on the main page in my local build. Is that expected?

**Comment by kimsauce** [comment]
> @kimsauce, I don't see the Google Translate widget on the main page in my local build. Is that expected?

It doesn't build on the dev side - production only - due to CORS.

**Comment by kimsauce** [comment]
@jpipkin1 Thank you so much for these fixes!! Would you be able to review once more?

**Comment by jpipkin1** [comment]
Looks great, @kimsauce1

**Review by jpipkin1 — APPROVED** [review]

**Comment by kimsauce** [comment]
> Looks great, @kimsauce!

thank you for the quick review

**Pull request merged**